FitVid/src/Components/Video/index.js

```diff
@@ -0,0 +1,18 @@
+import ReactPlayer from "react-player/lazy";
+import { useLiked } from "../../Context/liked-context";
+import { findIfVideoExistsInArray } from "../../Util/find-if-video-exists-in-array";
+import { addToLikedVideos } from "../../Util/add-to-liked";
+import { removeFromLikedVideos } from "../../Util/remove-from-liked";
+import { useToast } from "../../Context/toast-context";
+import { useUser } from "../../Context/user-context";
+import { useNavigate } from "react-router-dom";
+export {
```

Comment:
suggestion: we can directly export instead of importing and exporting, like 
export { useNavigate } from "react-router-dom";

Reply:
Yes we can but i like it this way